src/Model/iCalExport.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (C) 2017
+ *
+ *
+ *
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package Model;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Date;
+
+import biweekly.Biweekly;
+import biweekly.ICalendar;
+import biweekly.component.VEvent;
+import biweekly.property.Summary;
+import biweekly.util.Duration;
+
+/**
+ * @author amila
+ *
+ */
+public class iCalExport {
+	
+	private Date eStart;
+	private String title;
+	private MultilineString description;
+	private int hours;
+	private int minutes;
+	private int counter;
+	
+	public void createExportEvent(Event event, int counter){
+		seteStart(event.getDate());
+		setTitle(event.getName());
+		setDescription(event.getDetails());
+		this.counter = counter;
+		
+		
+		File file = new File("calendarExport/" + createFileName(counter));
+		
+		ICalendar ical = new ICalendar();
+		VEvent calEvent = new VEvent();
+		
+		Summary summary = calEvent.setSummary(title);
+		summary.setLanguage("en-us");
+		
+		//Default hard coded duration
```

Comment:
The default duration should not be set from hard coded numbers.  Constants (i.e., final static, whether public or private) should be declared and then used here.  Additionally, the default should likely be a duration of zero.  Your if-else below does not handle Activity or Deadline types, so a default of zero would be more sensible for something like a deadline that has no duration.